x/epochs/keeper/abci_test.go

```diff
@@ -10,146 +10,98 @@ import (
 	simapp "github.com/osmosis-labs/osmosis/v7/app"
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
-func TestEpochInfoChangesBeginBlockerAndInitGenesis(t *testing.T) {
-	var app *simapp.OsmosisApp
-	var ctx sdk.Context
-	var epochInfo types.EpochInfo
+	"golang.org/x/exp/maps"
 
-	now := time.Now()
+	"github.com/osmosis-labs/osmosis/v7/osmoutils"
+)
 
-	tests := []struct {
-		expCurrentEpochStartTime   time.Time
-		expCurrentEpochStartHeight int64
-		expCurrentEpoch            int64
-		expInitialEpochStartTime   time.Time
-		fn                         func()
+// This test is responsible for testing how epochs increment based off
+// of their initial conditions, and subsequent block height / times.
+//
+// TODO: Make a new test for init genesis logic
+func (suite KeeperTestSuite) TestEpochInfoBeginBlockChanges() {
+	block1Time := time.Unix(1656907200, 0).UTC()
+	// We run all tests with the same identifier and same duration
+	defaultIdentifier := "hourly"
+	defaultDuration := time.Hour
```

Comment:
What is `eps`? I'm assumming "epsilon"? Can we add a comment to explain why we need this?

Reply:
Why is this this value for epsilon is significant enough for a tick? From what I see, this is an implementation detail of the `time` package used for comparing time. Can we add a comment explaining this please?

Reply:
This comment is also slightly off for some reason. Referring to line 27

Reply:
Eps is epsilon. Its for an insignifcant time difference, its not a detail of the time package, but for bounds checking.

Reply:
(Nanosecond was the smallest time that my auto-completer could fill in. Could really be any unit of time)

Reply:
Thanks for clarifying. I think converting this into a comment would be useful :+1: